src/main/java/weizberg/citibike/StationMethods.java

```diff
@@ -0,0 +1,95 @@
+package weizberg.citibike;
+
+
+import io.reactivex.rxjava3.core.Single;
+import io.reactivex.rxjava3.schedulers.Schedulers;
+import weizberg.citibike.json.DataCollection;
+import weizberg.citibike.json.Station;
+
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+
+public class StationMethods {
+    private int numDocksAvailable;
+    private int numBikesAvailable;
```

Comment:
What is this?

Reply:
Please reply to this comment

Reply:
I am so sorry, I didn't see that comment before. I just removed it as it was unnecessary.